addons/event/models/event_ticket.py

```diff
@@ -82,12 +82,13 @@ def _compute_is_expired(self):
             else:
                 ticket.is_expired = False
 
-    @api.depends('start_sale_date', 'end_sale_date', 'event_id.date_tz')
+    @api.depends('start_sale_date', 'end_sale_date', 'event_id.date_tz', 'seats_available', 'seats_max')
     def _compute_sale_available(self):
         for ticket in self:
             current_date = fields.Date.context_today(ticket.with_context(tz=ticket._get_ticket_tz()))
             if (ticket.start_sale_date and ticket.start_sale_date > current_date) or \
-                    ticket.end_sale_date and ticket.end_sale_date < current_date:
+                    ticket.end_sale_date and ticket.end_sale_date < current_date or \
+                    ticket.seats_available <= 0 and ticket.seats_max > 0:
```

Comment:
Unless I missed something, sale_available is based on nothing else but:  
- start_sale_date and end_sale_date
- whether or not the associated product is active in event_sale

It allows me to make the difference between a ticket which is expired and a ticket which is sold_out. Useful in front-end to display info accordingly. 

I guess I could have used seats_available directly in front-end (have to check if some other computes didn't mess up because of that) but anyway, I thought it made little sense to let a sold out ticket have a sale_available as True